src/Slices/ServiceInventory/UI/Components/ForceStateAction.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useState } from "react";
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  Text,
+} from "@patternfly/react-core";
+import { CaretDownIcon } from "@patternfly/react-icons";
+import styled from "styled-components";
+import { Maybe, VersionedServiceInstanceIdentifier } from "@/Core";
+import { ActionDisabledTooltip } from "@/UI/Components";
+import { DependencyContext } from "@/UI/Dependency";
+import { words } from "@/UI/words";
+import ConfirmationModal from "./ConfirmationModal";
+import { ToastAlertMessage } from "./ToastAlertMessage";
+
+interface Props extends VersionedServiceInstanceIdentifier {
+  instance_identity: string;
+  targets: string[];
```

Comment:
What are the targets actually ?

Reply:
possible instance  states, I'll change name as it not self-explanatory